### PR TITLE
OSD-14689: Add backplane-cli create and get testjob commands

### PR DESCRIPTION
**Pull request by samanthajayasinghe**
### What type of PR is this?

feature

### What this PR does / Why we need it?
Adding test job create and get commands 

### Which Jira/Github issue(s) does this PR fix?

_Resolves #_ [OSD-14689](https://issues.redhat.com/browse/OSD-14689)

### Pre-checks (if applicable)

- [x] Ran unit tests locally
- [x] Validated the changes in a cluster
- [ ] Included documentation changes with PR

### CODE REVIEW

File: cmd/ocm-backplane/testJob/testJob.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import (
+	"github.com/spf13/cobra"
+)
+
+func NewTestJobCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "testjob",
+		Aliases:      []string{"testJob", "testjobs", "tj"},
+		Short:        "Represents a backplane testJob.",
+		SilenceUsage: true,
```
Comment (wanghaoran1988) [inline cmd/ocm-backplane/testJob/testJob.go:12]: Let's hide this command 
```suggestion
		SilenceUsage: true,
		Hidden:       true,
```

Comment (samanthajayasinghe) [reply cmd/ocm-backplane/testJob/testJob.go:12]: done

**Pull request merged**